assets/src/edit-story/components/library/panes/text/textPresets.js

```diff
@@ -88,6 +103,30 @@ const PRESETS = [
       width: DEFAULT_ELEMENT_WIDTH,
     },
   },
+  {
+    title: __('Caption', 'web-stories'),
+    element: {
+      content: __('Caption', 'web-stories'),
+      fontSize: dataFontEm(1),
+      lineHeight: 1.5,
+      x: DEFAULT_LEFT_MARGIN,
+      y: (PAGE_HEIGHT - dataFontEm(1)) / 2,
+      font: TEXT_ELEMENT_DEFAULT_FONT,
+      width: DEFAULT_ELEMENT_WIDTH,
+    },
+  },
+  {
+    title: __('OVERLINE', 'web-stories'),
+    element: {
+      content: __('OVERLINE', 'web-stories'),
```

Comment:
Ideally these strings aren't capitalized themselves but styled via CSS. Something for the next iteration.